### PR TITLE
Issues 32 info cards

### LINKED ISSUE #32: Info Cards
image
https://www.vonage.com/unified-communications/integrations/

### PR DESCRIPTION
**Pull request by david-granberg**
Fix #32

Test URLs:
- Before: https://main--vonage--hlxsites.hlx.page/unified-communications/integrations
- After: https://issues-32-info-cards--vonage--hlxsites.hlx.page/unified-communications/integrations

Scroll down to see the new block variant in action or have a look directly at my test page:
http://localhost:3000/drafts/dgranber/black-gradient-highlight-card

No special call outs that I can think of, hopefully a quick win to get merged in :D

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -149,3 +186,245 @@
...
+
+  /* Div to contain the card elements */
+  .cards.info-card .card-container {
+    padding-left: 20px;
+    padding-right: 20px;
+    cursor: pointer;
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
```
Comment (synox) [inline blocks/cards/cards.css:224]: These -webkit prefixes are usually not needed anymore. please try if it works without.

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -149,3 +186,245 @@
...
+    font-weight: 400;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  /* Styling of the arrow icon and it's animation */
+  .cards.info-card .icon-arrow-link::before {
+    content: var(--vlt-right-arrow);
+    color: var(--info-card-color);
+    font-family: VltIcons!important;
```
Comment (synox) [inline blocks/cards/cards.css:318]: there is now a variable for this font

File: blocks/cards/cards.js [hunk 3]
```diff
@@ -24,6 +24,41 @@ export default function decorate(block) {
...
+      // Pull out the <a> tags from the existing markdown
+      cardDivWrapper.querySelector('h2').removeChild(cardDivWrapper.querySelector('h2 > a'));
+      cardDivWrapper.querySelector('p').removeChild(cardDivWrapper.querySelector('p > a'));
+
+      cardDivWrapper.querySelector('h2').innerText = titleText;
+      cardDivWrapper.querySelector('p').innerText = buttonText;
+
+      // Copy the div HTML contents as the a contents and
+      // replace them in the parent container (the li)
+      cardLinkWrapper.innerHTML = cardDivWrapper.innerHTML;
```
Comment (synox) [inline blocks/cards/cards.js:54]: Try if it also works with this, would be a bit cleaner:  
```
cardLinkWrapper.append(...cardDivWrapper.children)
```

File: styles/styles.css [hunk 4]
```diff
@@ -25,6 +25,9 @@
   --heading-font-family: var(--body-font-family);
   --fixed-font-family: "Spezia_SemiMono", menlo, consolas, 'Liberation Mono', monospace;
 
+   /* font icon aliases */
+   --vlt-right-arrow: "\e90c";
```
Comment (synox) [inline styles/styles.css:29]: good idea! 👏

File: blocks/cards/cards.css (hunk 1 above)
Comment (david-granberg) [reply blocks/cards/cards.css:324]: Sounds good, yeah think that was some cruft that I brought in while copying over the styles, I'll make a note to test without going forward.

Just tested pulling those props out and it did indeed make no difference to the rendering in browser, thanks!

File: blocks/cards/cards.css (hunk 2 above)
Comment (david-granberg) [reply blocks/cards/cards.css:406]: Groovy!  Will find and update my instances using it

Comment (david-granberg) [reply blocks/cards/cards.css]: Whelp I added my own spin on the variable to this branch (given that the one you mentioned is not in main yet), if you get your PR merged in first I'll rebase

File: styles/styles.css (hunk 4 above)
Comment (david-granberg) [reply styles/styles.css:30]: Yup!  If I get a moment I can find all the instances of icons in the font pack, and generate suitable human readable names for all of them to make them easier to leverage going forward (maybe will spin off a quick branch to do that)

**Review by synox — APPROVED** [review]
looks good. Just a minor ask to avoid .innerHTML

File: blocks/cards/cards.js (hunk 3 above)
Comment (david-granberg) [reply blocks/cards/cards.js]: Yup, using the spread operator worked like a charm.  Good tip!  I'll tuck that away in my bag of JS tricks that are slowly getting more modern ;) 

Thanks!

**Pull request merged**